coding-concepts.js

```diff
@@ -5,47 +5,50 @@
 // First, predict the outcome based on your understanding of the code.
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
-// --------------------1) What will this log?
+// --------------------1) What will this log? 
+
+
 
 var cohort = "Bravo 2021"
-// console.log(cohort.length)
+ console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:It will log Bravo 2021.
+// b) Verify and explain:I did not pay atention to .length so the actual out put was 10
 
 
 // --------------------2) What will this log?
+//This will log l
 
 var greeting = "Hello World!"
-// console.log(greeting[3])
+ console.log(greeting[3])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: This will log l
+// b) Verify and explain: I will log l because it is asking for the index of 3 
 
 
 // --------------------3) What will this log?
 
 var languages = ["JavaScript", "Ruby", "Python", "C++"]
 var index = 1
-// console.log(languages[index])
+ console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I will log ["JavaScript", "Ruby", "Python", "C++"] and the 1
+// b) Verify and explain: I was worng, the it logged Ruby. I dont understand why 
```

Comment:
Even though the console.log is `languages[index]` the index is a variable holding the number 1 so we can read this as `languages[1]`